pageserver/src/http/openapi_spec.yml

```diff
@@ -257,6 +257,37 @@ paths:
               schema:
                 $ref: "#/components/schemas/LsnByTimestampResponse"
 
+  /v1/tenant/{tenant_id}/timeline/{timeline_id}/lsn_lease:
```

Comment:
nit: For new APIs, we can use `tenant_shard_id` as the parameter name (it's only left as tenant_id for other APIs as a backward compatibility measure for clients that use codegen)

Reply:
done